src/planner.c

```diff
@@ -252,31 +136,17 @@ timescaledb_planner(Query *parse, int cursor_opts, ParamListInfo bound_params)
 		ts_cache_release(hc);
 	}
 
-
 	if (prev_planner_hook != NULL)
-	{
 		/* Call any earlier hooks */
-		plan_stmt = (prev_planner_hook) (parse, cursor_opts, bound_params);
-	}
-	else
-	{
-		/* Call the standard planner */
-		plan_stmt = standard_planner(parse, cursor_opts, bound_params);
-	}
+		return (prev_planner_hook) (parse, cursor_opts, bound_params);
 
-	if (ts_extension_is_loaded())
-	{
-		ModifyTableWalkerCtx ctx = {
-			.parse = parse,
-			.hcache = ts_hypertable_cache_pin(),
-			.rtable = plan_stmt->rtable,
-		};
-
-		ts_planned_stmt_walker(plan_stmt, modifytable_plan_walker, &ctx);
-		ts_cache_release(ctx.hcache);
-	}
+	/* Call the standard planner */
+	stmt = standard_planner(parse, cursor_opts, bound_params);
 
-	return plan_stmt;
+	/* Fixup top-level target list for inserts */
```

Comment:
please explain more or add a pointer to comment in hypertable_insert_plan_create.